tests/playbooks/e2e.yaml

```diff
@@ -93,7 +94,29 @@
         name: '{{ vm_name }}'
         namespace: default
 
-    - name: Incrase the resources of VM
+    - name: Wait for VMI status to running
+      k8s_facts:
+        name: "{{ vm_name }}"
+        namespace: default
+        api_version: kubevirt.io/v1alpha2
+        kind: VirtualMachineInstance
+      register: vmi
+      until: "vmi.resources[0].status.phase == 'Running'"
+      delay: 10
+      retries: 18
+
+    - name: Wait for VM to be ready
+      k8s_facts:
+        name: "{{ vm_name }}"
+        namespace: default
+        kind: VirtualMachine
+        api_version: kubevirt.io/v1alpha2
+      register: vm
+      until: "vm.resources[0].status['created'] == true and vm.resources[0].status['ready'] == true"
```

Comment:
Hmm, we don't have this in wait for running state in `kubevirt_vm` module, I think we should add it.

Reply:
@machacekondra are you going to add this to kubevirt_vm or should I keep it?

Reply:
Keep it for now, I will remove it once we add it to `kubevirt_vm`.